modules/postgres/resources/customEntrypoint.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -Eeo pipefail
+
+
+pUID=$(id -u postgres)
+pGID=$(id -g postgres)
+
+if [ -z "$pUID" ]
+then
+    exit 1
+fi
+
+if [ -z "$pGID" ]
+then
+    exit 1
+fi
+
+chown "$pUID":"$pGID" /tmp/data/ca_cert.pem
+chown "$pUID":"$pGID" /tmp/data/server.cert
+chown "$pUID":"$pGID" /tmp/data/server.key
```

Comment:
Would this be enough?
```suggestion
chown -R "${pUID}:${pGID}" /tmp/data
```